src/zenml/cli/utils.py

```diff
@@ -970,6 +970,42 @@ def print_components_table(
     print_table(configurations)
 
 
+def print_debug_stack(
+    stack: "StackResponseModel", components: "List[ComponentResponseModel]"
+) -> None:
+    """Print stack and components for debugging purposes.
+
+    Args:
+        stack: A stack
+        components: List of components
+    """
+    declare("\nCURRENT STACK\n", bold=True)
+    console.print(f"Name: {stack.name}")
+    console.print(f"ID: {str(stack.id)}")
+    console.print(f"Shared: {'Yes' if stack.is_shared else 'No'}")
+    console.print(f"User: {stack.user.name} / {str(stack.user.id)}")
+    console.print(
+        f"Workspace: {stack.workspace.name} / {str(stack.workspace.id)}"
+    )
+
+    for component in components:
+        declare(
+            f"\n{component.type.value.upper()}: {component.name}\n", bold=True
+        )
+        console.print(f"Name: {component.name}")
```

Comment:
Is this essentially a better looking version of `zenml stack-component describe`? If yes, can we repurpose it for that somehow?

Reply:
Not quite. I explicitly didn't want all the rich tabular formatting so that it was copy-pastable, so not sure how much crossover there is any more then.